SolarSystem.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE NegativeLiterals #-}
+
+import CodeWorld
+
+main :: IO ()
+main = animationOf solarSystem
+
+solarSystem :: (Double -> Picture)
+solarSystem = 
+  \t ->
+    pictures [colored yellow (solidCircle 1),
+    translated (cos (t/0.24) * 2) (sin (t/0.24) * 2)(colored (mixed [brown, grey]) (solidCircle 0.15)),
```

Comment:
It's not easy to guess what these coefficients are. What is `0.24`, what is `2`? Perhaps, create a helper data type that describes a planet, and a function to draw it?

```
data Planet = MkPlanet { size :: ..., color :: ..., period: ... }

planet :: Planet -> Double -> Picture
```

Reply:
Good idea, thanks!